packages/sources/aleno/package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Can you run `yarn` as well so that dependencies are pulled and the CI will work?

Reply:
done, issue was that typescript and fw versions were incorrect, no new deps